data.sql

```diff
@@ -12,3 +12,19 @@ VALUES
   ('Pikachu', '2021-01-07', 1, false, 15.04),
   ('Devimon', '2017-05-12', 5, true, 11);
 
+INSERT INTO
+  animals (
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES
+  ('Charmander', '2020-02-08', 0, false, 11),
+  ('Plantmon', '2021-11-15', 2, true, 5.7),
+  ('Squirtle', '1993-04-02', 3, false, 12.13),
+  ('Angemon', '2005-06-12', 1, true, 45),
+  ('Boarmon', '2005-06-07', 7, true, 20.4),
+  ('Blossom', '1998-10-13', 3, true, 17),
+  ('Ditto', '2022-05-14', 4, true, 22);
```

Comment:
Kindly make sure that you include the negative weights for `Charmander`, `Plantmon`, `Squirtle`, and `Angemon` as in the project requirements. The negative weights are not a mistake, they are there on purpose.

They will later be corrected with the transaction query in the `querries.sql` file:
```sql
    UPDATE animals SET weight_kg = weight_kg * -1; 
    ROLLBACK TO SP1;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; 
```